bouncer/shared/upgrade_network.ts

```diff
@@ -350,5 +351,9 @@ export async function upgradeNetworkPrebuilt(
     await incompatibleUpgradeNoBuild(localnetInitPath, binariesPath, runtimePath, numberOfNodes);
   }
 
+  if (cleanOldVersion === '1.6.1') {
```

Comment:
We might bump the versions... as we have, we're at 1.6.4 now

I believe you can do something like
```suggestion
  if (cleanOldVersion.startsWith("1.6")) {
```